### PR TITLE
problem: solution field is not displayed #60

### PR DESCRIPTION
**Pull request by samuelralak**
Solution: Added solution to problem detail page

### CODE REVIEW

**Comment by gazhayes** [comment]
@samuelralak conflicts

**Comment by samuelralak** [comment]
on it.

**Comment by samuelralak** [comment]
fixed conflicts. @geekyd let me know if PR overwrites any of your changes once merged

File: imports/ui/components/documents/show/document-show.html [hunk 1]
```diff
@@ -14,8 +14,20 @@
           {{#if problem.claimed}}
           <span class="badge badge-success">CLAIMED</span>
           {{/if}}
-          <h2>{{problem.summary}}</h2>
-          <p>{{problem.description}}</p>
+
+
+          {{#if isProblemOwner problem.createdBy}}
```
Comment (geekyd) [inline imports/ui/components/documents/show/document-show.html]: @samuelralak These wont be required, I have already added the buttons. 
Please check 
https://github.com/EmurgoHK/Emurgis/blob/master/imports/ui/components/documents/show/document-show.html#L9

Comment (samuelralak) [reply imports/ui/components/documents/show/document-show.html]: I had done that as part of [#63](https://github.com/EmurgoHK/Emurgis/issues/63). not to worry though, my next PR should fix all that will have been deleted

**Pull request merged**